physics/cam_diagnostics.F90

```diff
@@ -365,6 +379,364 @@ subroutine diag_init_dry(pbuf2d)
+      call add_default ('BS550AER', 1, ' ')
+      !
+      call add_default ('EC550SO4', 1, ' ')
+      call add_default ('EC550BC ', 1, ' ')
+      call add_default ('EC550POM', 1, ' ')
+      call add_default ('EC550SS ', 1, ' ')
+      call add_default ('EC550DU ', 1, ' ')
+      !
+      call add_default ('CDOD440 ', 1, ' ')
+      call add_default ('CDOD550 ', 1, ' ')
+      call add_default ('CABS550 ', 1, ' ')
+      call add_default ('CABS550A', 1, ' ')
+      call add_default ('CDOD870 ', 1, ' ')
+      call add_default ('A550_DU ', 1, ' ')
+      call add_default ('A550_SS ', 1, ' ')
+      call add_default ('A550_SO4', 1, ' ')
+      call add_default ('A550_POM', 1, ' ')
+      call add_default ('A550_BC ', 1, ' ')
+      call add_default ('D440_DU ', 1, ' ')
+      call add_default ('D440_SS ', 1, ' ')
+      call add_default ('D440_SO4', 1, ' ')
+      call add_default ('D440_POM', 1, ' ')
+      call add_default ('D440_BC ', 1, ' ')
+      call add_default ('D500_DU ', 1, ' ')
+      call add_default ('D500_SS ', 1, ' ')
+      call add_default ('D500_SO4', 1, ' ')
+      call add_default ('D500_POM', 1, ' ')
+      call add_default ('D500_BC ', 1, ' ')
+      call add_default ('D550_DU ', 1, ' ')
+      call add_default ('D550_SS ', 1, ' ')
+      call add_default ('D550_SO4', 1, ' ')
+      call add_default ('D550_POM', 1, ' ')
+      call add_default ('D550_BC ', 1, ' ')
+      call add_default ('D670_DU ', 1, ' ')
+      call add_default ('D670_SS ', 1, ' ')
+      call add_default ('D670_SO4', 1, ' ')
+      call add_default ('D670_POM', 1, ' ')
+      call add_default ('D670_BC ', 1, ' ')
+      call add_default ('D870_DU ', 1, ' ')
+      call add_default ('D870_SS ', 1, ' ')
+      call add_default ('D870_SO4', 1, ' ')
+      call add_default ('D870_POM', 1, ' ')
+      call add_default ('D870_BC ', 1, ' ')
+      call add_default ('DLT_DUST', 1, ' ')
+      call add_default ('DLT_SS  ', 1, ' ')
+      call add_default ('DLT_SO4 ', 1, ' ')
+      call add_default ('DLT_POM ', 1, ' ')
+      call add_default ('DLT_BC  ', 1, ' ')
+      call add_default ('DGT_DUST', 1, ' ')
+      call add_default ('DGT_SS  ', 1, ' ')
+      call add_default ('DGT_SO4 ', 1, ' ')
+      call add_default ('DGT_POM ', 1, ' ')
+      call add_default ('DGT_BC  ', 1, ' ')
+      call add_default ('NNAT_0  ', 1, ' ')
+      call add_default ('NNAT_1  ', 1, ' ')
+      call add_default ('NNAT_2  ', 1, ' ')
+      call add_default ('NNAT_4  ', 1, ' ')
+      call add_default ('NNAT_5  ', 1, ' ')
+      call add_default ('NNAT_6  ', 1, ' ')
+      call add_default ('NNAT_7  ', 1, ' ')
+      call add_default ('NNAT_8  ', 1, ' ')
+      call add_default ('NNAT_9  ', 1, ' ')
+      call add_default ('NNAT_10 ', 1, ' ')
+      call add_default ('NNAT_12 ', 1, ' ')
+      call add_default ('NNAT_14 ', 1, ' ')
+      call add_default ('AIRMASSL', 1, ' ')  !akc6
+      call add_default ('AIRMASS ', 1, ' ')  !akc6
```

Comment:
Does anyone know what those comments are?

Reply:
I'm simply going to remove them.

Reply:
I'm been removing these type of comments as I go. It makes no sense at this point to keep them in.